### ISSUE TITLE
add "listen" or "play" button to button cluster

### ISSUE THREAD

**Issue by echarlie** [wuvt/pload#38]
this would open the link to the track in a new tab (and thus start playing with in-browser player), and be easier than clicking on the hamburger and "open in new tab" on the link.

**Comment by mutantmonkey** [comment]
I was actually thinking about adding an integrated player into the track info modal instead, but this could work too.

**Comment by echarlie** [comment]
this would be a simpler/faster way to do it, I think. we can add the integrated player later.

**Comment by mutantmonkey** [comment]
Indeed. For an integrated player, we may also need to add yet another rewrite for "public" URLs since at least in the case of wuvtLinx, the display URL points to the preview page. We could also simply remove that and require that the display URLs be a direct link to the media.
